us_hate_crimes.md

```diff
@@ -32,7 +32,7 @@ _Definition: a prejudice-motivated crime which occurs when a perpetrator targets
 - On August 14th, 2017, at the [Unite the Right rally](https://en.wikipedia.org/wiki/Unite_the_Right_rally) in Charlottesville VA, 20-year old Neo-nazi James Alex Fields Jr., [drove his car into a crowd of antifascist counterprotestors](https://en.wikipedia.org/wiki/Unite_the_Right_rally#Vehicular_attack_and_homicide), wounding 19, and killing Heather Heyer, a paralegal from Charlottesville. Heyer's mother said she wanted Heather's name to become "a rallying cry for justice and equality and fairness and compassion." <sup>[1](https://en.wikipedia.org/wiki/Unite_the_Right_rally#Vehicular_attack_and_homicide)</sup>
 - On August 12, 2017, [DeAndre Harris was assaulted by six white men in an attack](https://en.wikipedia.org/wiki/Assault_of_DeAndre_Harris) in a parking garage next to the police headquarters during the [Unite the Right rally](https://en.wikipedia.org/wiki/Unite_the_Right_rally) in [Charlottesville, Virginia](https://en.wikipedia.org/wiki/Charlottesville,_Virginia), United States. Images and video of the assault captured by photojournalist Zach Roberts went viral and became a symbol of the enmity underlying the protest.
 - On August 5, 2017, three members of a far-right militia [bombed the Dar Al-Farooq Islamic Center in Bloomington, Minnesota.](https://www.nytimes.com/2017/08/05/us/minnesota-mosque-explosion.html) McWhorter is alleged to have told an FBI agent that the attack was an attempt “to scare” Muslims “out of the country.” Luckily no one was killed. Back in 2017, one of the bombers, Hari, who owns a security company, [submitted a $10 billion proposal](https://www.washingtonpost.com/news/post-nation/wp/2018/03/14/he-wanted-to-build-trumps-border-wall-he-was-just-charged-in-the-bombing-of-a-minnesota-mosque/?utm_term=.f9c1d8da2fe1) to build Trump’s wall along the U.S.-Mexico border. “We would look at the wall as not just a physical barrier to immigration but also as a symbol of the American determination to defend our culture, our language, our heritage, from any outsiders,” Hari said.
-- On July 14th, 2017, alt-right activist, anti-feminist, and former intern of Milo Yiannapoulous, Lane Davis, [murdered his father for calling him a nazi](https://www.thedailybeast.com/youtube-trumpkin-and-former-milo-intern-kills-his-own-dad-for-calling-him-a-nazi).
+- On July 14th, 2017, alt-right activist, anti-feminist, and former intern of Milo Yiannapoulous, Lane Davis, [murdered his father for calling him a nazi](https://www.thedailybeast.coyoutube-trumpkin-and-former-milo-intern-kills-his-own-dad-for-calling-him-a-nazi).
```

Comment:
Check this one out. What was the regex / replacer you used?

Reply:
Well spotted, I forgot to escape the `.` in a rushed `sed -i "s/m.youtube/youtube/g" *.md`

I can't spot any others impacted by that one overreaching so I've just force-pushed a correction.

Reply:
Sweet, thx.